federatedscope/core/auxiliaries/logging.py

```diff
@@ -158,8 +158,8 @@ def init_wandb(cfg):
                entity=cfg.wandb.name_user,
                config=cfg_yaml,
                group=dataset_name,
-               job_type=method_name,
-               name=exp_name,
+               job_type=exp_name,
```

Comment:
Why do we need to exchange `exp_name` and `method_name`?

Reply:
this is only related to the viewing logging in wandb web UI. in my exps (I believe applicable to most LLMs experiments) generally the config yaml is modified but the optimizer (corresponding to method_name) is not. In the wandb API, the job_type Is listed as separate colored experiments where the different names are different runs of the same color group. If the changes seem unnecessary, they can be reverted.

Reply:
This change will affect some historical experiments (e.g. scripts in our benchmarks). IMHO, it only affects the presentation of the results, but not the correctness of the results. That is, the user will see the grouped results on the WandB page in a way that is inconsistent with some historical result. If the user wants to change it, the cost should be very small.

In general I agree with this change, but a comment could be added to simply remind the user, e.g. "in [PR#xx]](link goes here), the `exp_name` and `method_name` were exchanged for a better grouping manner"